examples/monitors-logging/main.tf

```diff
@@ -3,7 +3,7 @@ resource "chronosphere_monitor" "monitor_with_logging_query" {
   collection_id = chronosphere_collection.infra.id
   query {
     logging_expr = <<-EOF
-      service = "nginx" | make-series by chronosphere_namespace
+      service = "nginx" | make-series by chronosphere_namespace step 1m
```

Comment:
is this a fix for something else?

Reply:
Yes, seems like logging monitors made step a required thing, causing scenarios to fail.